crates/mun_codegen/src/snapshots/test__main.snap

```diff
@@ -0,0 +1,58 @@
+---
+source: crates/mun_codegen/src/test.rs
```

Comment:
This test name seems incorrect for the included tests. Was this added by mistake?